notes/points-markets-settlement.md

```diff
@@ -0,0 +1,103 @@
+
+The approach is modelled on Credit Default Swaps (CDS) which are derivatives that are valued based on the probability of a company defaulting. Payouts are triggered when a “credit event” occurs, as defined by a set of explicit rules. Analogously, we define the concept of an “airdrop event” that can be similarly used to determine when and at what value to settle points futures.
+
+
+## Conceptual overview
+
+Conceptually, we define three things to enable fair and orderly settlement of a points futures market:
+
+1. The conditions under which the ***Airdrop Event*** has been deemed to have occurred, and the market may settle.
+
+2. The rules for when to halt trading prior to settlement (to ensure orderly settlement while the final price is calculated and verified) due to an ***Imminent Airdrop Event***.
+
+3. The ***Valuation Algorithm*** for calculating the *mean conversion rate* that defines the value of one point at settlement.
+
+
+## Detailed definitions
+
+### Terms
+
+These terms have specific meanings in the text below
+
+*points*
+: Any pseudo-asset issued by a protocol or company (e.g. for activity or engagement with a project/protocol/product) that is not fungible or directly tradable. For example, points issued by protocols/networks like Hyperliquid, EigenLayer as well as loyalty points such as those issued by airlines and supermarkets.
+
+*conversion*
+: Any process by which some or all *points* become fungible assets such as protocol tokens or stablecoins, or become definitively valueless (e.g. a user/key’s balance is cleared). *Conversion* is considered to be complete when it is possible to value the points involved in the `Base Asset` (i.e. if points convert to a token with no market price, conversion may not be complete).
+
+*mean conversion rate*
+: The average rate in terms of the `Base Asset` at which each point involved in an *Airdrop Event* underwent *conversion*.
+
+*termination date/time*
+: The time at which trading on the market should stop as a result of an ***Imminent Airdrop Event***.
+
+
+### Variables
+
+The following terms (with suggested values in brackets) used in the definitions refer to quantities or other parameters that affect the specifics of the product. They may be varied to suit each individual use case.
+
+`Base Asset` (USDT)
+: The asset in which *points* that have undergone `conversion` are valued.
+
+`Inclusivity Threshold` (40.00%)
+: The percentage of issued points that must be eligible for the conversion
+
+`Liquidity Threshold` (5.00%)
+: The percentage of issued points that must be met by the bid depth (quantity of resting buy orders or points otherwise able to be sold) in the market within `valid slippage` (%) of the current market price.
```

Comment:
Further to @TomMcL's point about "which market" below. Here we say "the market" - is it worth clarifying here already?

Reply:
"the current price per point" may be more clear